build-tools/conjure-xamarin-android-cecil/conjure-xamarin-android-cecil.cs

```diff
@@ -10,10 +10,11 @@ public class ConjureXamarinAndroidCecil
 	const string BaseNameReplacement = "Xamarin.Android.Cecil";
 	const string CecilAssemblyName = BaseNameReplacement;
 	const string CecilMdbAssemblyName = BaseNameReplacement + ".Mdb";
+	const string PublicKey = "024004800094000620002400525341310400101079159977d2d03a8e6bea7a2e74e8d1afcc93e8851974952bb480a12c9134474d462447c37ee68c080536fcf3c3fbe2ff9c979ce998475e56e8ce82dd5bf35dc1e93bf2eeecf874b2477c5081dbea7447fddafa277b22de47d6ffea449674a4f9fccf84d1506989380284dbdd35f46cdff12a1bd78e4ef065d016df";
```

Comment:
I'm not sure that this `PublicKey` is correct either.  If I take my local build -- *or* look at the installers created by [this PR](https://dev.azure.com/DevDiv/DevDiv/_build/results?buildId=2882865&view=artifacts) -- then I see the following:

```bash
$ monodis --assembly bin/Debug/lib/xamarin.android/xbuild/Xamarin/Android/Xamarin.Android.Cecil.Mdb.dll
Assembly Table
Name:          Xamarin.Android.Cecil.Mdb
Hash Algoritm: 0x00008004
Version:       0.11.0.0
Flags:         0x00000001
PublicKey:     BlobPtr (0x00000001)
	Dump:
0x00000000: 00 24 00 00 04 80 00 00 94 00 00 00 06 02 00 00 
0x00000010: 00 24 00 00 52 53 41 31 00 04 00 00 01 00 01 00 
0x00000020: 79 15 99 77 D2 D0 3A 8E 6B EA 7A 2E 74 E8 D1 AF 
0x00000030: CC 93 E8 85 19 74 95 2B B4 80 A1 2C 91 34 47 4D 
0x00000040: 04 06 24 47 C3 7E 0E 68 C0 80 53 6F CF 3C 3F BE 
0x00000050: 2F F9 C9 79 CE 99 84 75 E5 06 E8 CE 82 DD 5B 0F 
0x00000060: 35 0D C1 0E 93 BF 2E EE CF 87 4B 24 77 0C 50 81 
0x00000070: DB EA 74 47 FD DA FA 27 7B 22 DE 47 D6 FF EA 44 
0x00000080: 96 74 A4 F9 FC CF 84 D1 50 69 08 93 80 28 4D BD 
0x00000090: D3 5F 46 CD FF 12 A1 BD 78 E4 EF 00 65 D0 16 DF 
Culture:       

```

Notice that it starts with `00 02`, not `024`.  I think this string is missing a leading `0`.

I also think it's *not long enough*.  `monodis --asssembly` shows a *160* byte PublicKey, which should be 160*2=320 characters in length, yet *this* string is only 288.  I think it should instead be:

```
const string PublicKey =
	"00240000048000009400000006020000" +
	"00240000525341310004000001000100" +
	"79159977D2D03A8E6BEA7A2E74E8D1AF" +
	"CC93E8851974952BB480A12C9134474D" +
	"04062447C37E0E68C080536FCF3C3FBE" +
	"2FF9C979CE998475E506E8CE82DD5B0F" +
	"350DC10E93BF2EEECF874B24770C5081" +
	"DBEA7447FDDAFA277B22DE47D6FFEA44" +
	"9674A4F9FCCF84D15069089380284DBD" +
	"D35F46CDFF12A1BD78E4EF0065D016DF";
```

This multiline string is also easier to verify against `monodis --assembly` output.

Reply:
Ok great it works now, thanks. ILSpy must not display the public key correctly, I pasted the values it was showing me into a diff tool and they matched...